RadixWallet/Clients/OnboardingClient/OnboardingClient+Interface.swift

```diff
@@ -1,22 +1,28 @@
 // MARK: - OnboardingClient
 public struct OnboardingClient: Sendable {
-	public var loadProfile: LoadProfile
+	/// Call this when user has finished authentication from lock screen (e.g. Splash)
+	public var unlockedApp: UnlockedApp // FIXME: Move to a new Lock/Unlock client?
 
-	/// Returns `true` iff Profile contains any mainnet accounts
-	public var commitEphemeral: CommitEphemeral
+	public var loadProfile: LoadProfile
+	public var finishedOnboarding: FinishedOnboarding
 
 	public init(
+		unlockedApp: @escaping UnlockedApp,
 		loadProfile: @escaping LoadProfile,
-		commitEphemeral: @escaping CommitEphemeral
+		finishedOnboarding: @escaping FinishedOnboarding
 	) {
+		self.unlockedApp = unlockedApp
 		self.loadProfile = loadProfile
-		self.commitEphemeral = commitEphemeral
+		self.finishedOnboarding = finishedOnboarding
 	}
 }
 
 extension OnboardingClient {
-	public typealias LoadProfile = @Sendable () async -> LoadProfileOutcome
+	public typealias LoadProfile = @Sendable () async -> Profile
+
+	public typealias FinishedOnboarding = @Sendable () async -> EqVoid
```

Comment:
Minor: should be `FinishOnboarding`? I mean it should describe an action to perform. Same for `UnlockApp`. or maybe other naming, like "handleFinishedOnboarding"

Reply:
yeah probably, will change

Reply:
fixed